src/commands/properties/verify.ts

```diff
@@ -94,8 +94,14 @@ export class PropertyVerifyCommand extends PropertiesUpdateCommand {
         const providerUNID = this.getServiceProviderUNID();
         const jwtToken = await new JWTVerifier(this.unsClientWrapper.unsClient).verifyUnsJWT(rawJwt, providerUNID);
 
+        const proof = JSON.stringify({
```

Comment:
`stringify` isn't strictly necessary here, as the object will be serialized from the CLI to the SUN API :wink: